package.json

```diff
@@ -60,6 +60,7 @@
     "is-promise": "^2.1.0",
     "lodash": "^4.12.0",
     "lodash-es": "^4.12.0",
+    "promise": "^7.1.1",
```

Comment:
Do you already use another promise library? Or do you mean the native Promises? In that case the users would need the poly fill to make sure it always works?